badger/cmd/write_bench.go

```diff
@@ -298,6 +301,32 @@ func writeBench(cmd *cobra.Command, args []string) error {
 	return err
 }
 
+func updateKeysStats(db *badger.DB) {
+	txn := db.NewTransaction(false)
+	defer txn.Discard()
+
+	iopt := badger.DefaultIteratorOptions
+	iopt.AllVersions = true
+	iopt.InternalAccess = true
+	it := txn.NewIterator(iopt)
+	defer it.Close()
+
+	for it.Rewind(); it.Valid(); it.Next() {
+		i := it.Item()
+		if bytes.HasPrefix(i.Key(), []byte("!badger!")) {
+			internalKeyCount++
```

Comment:
Move these variables inside this function and then print them at the end in this function itself. We don't want to accumulate all the counts.